.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2.1
+
+workflows:
+  main:
+    jobs:
+      - lint
+      - unit-tests
+      - build
+
+jobs:
+  lint:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run:
+          name: "Lint Code"
+          command: |
+            yarn install --frozen-lockfile
+            yarn lint
+  build:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run:
+          name: "Build"
+          command: |
+            yarn install --frozen-lockfile
+            yarn build
+  unit-tests:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run:
+          name: "Unit Tests"
+          command: |
+            yarn install --frozen-lockfile
```

Comment:
You could batch all of these into one job with multiple commands like how I did with Treeherder, then you don't need to install yarn each time.